runtime/lua/vim/uri.lua

```diff
@@ -101,6 +101,7 @@ local function uri_to_fname(uri)
   -- TODO improve this.
   if is_windows_file_uri(uri) then
     uri = uri:gsub('^file:///', '')
+    uri = uri:gsub('^([a-z]):', function(a,b) return string.upper(a) .. ":" .. b end)
```

Comment:
This errors out. There is only one capture group. Changing the line to the following, I confirm, fixes our lsp bug.

```suggestion
    uri = uri:gsub('^([a-z]):', function(a) return string.upper(a) .. ":" end)
```

Reply:
fixed, thanks!

Reply:
Yup this solves it <3